mmv1/products/privateca/CaPool.yaml

```diff
@@ -130,8 +130,7 @@ properties:
                 - !ruby/object:Api::Type::Enum
                   name: 'signatureAlgorithm'
                   description: |
-                    The algorithm used.
-                  required: true
+                    A signature algorithm that must be used. If this is omitted, any EC-based signature algorithm will be allowed.
```

Comment:
In this case, the `ellipticCurve` object can only have one possible field `signatureAlgorithm`, which I believe is why it is required here. Sometimes empty objects can be used, but it is rare. Would `elliptic_curve = {}` be valid configuration?

My guess is that `signatureAlgorithm` should remain required.

I noticed a similar pattern on some of the other fields as well, and wanted to make sure to clarify: the `required: true` means the field is required within the context of its parent. So for the other nested fields where you've removed `required: true`, I'd like to make sure that the parent object can still be used without those fields.

Reply:
Thanks for your detailed review. 

Technically, yes. `elliptic_curve = {}` is equivalent to `elliptic_curve = { UNSPECIFIED_ALGO }`, which means [any EC-based signature algorithm will be allowed](http://google3/google/cloud/security/privateca/v1/resources.proto;l=518-519;rcl=549523805). However, it seems the [proto validator](http://google3/google/cloud/security/privateca/v1/resources.proto;l=523;rcl=549523805) blocked that use case. I will work with the team to unblock it.

Similar changes for cl/553167307 .

I will update the PR once it is ready to move forward.